src/nikonmn_int.cpp

```diff
@@ -2578,15 +2561,9 @@ fmountlens[] = {
 
         byte raw[] = { 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0 };
 
-        static const char* tags[] = {
-            "LensIDNumber",
-            "LensFStops",
-            "MinFocalLength",
-            "MaxFocalLength",
-            "MaxApertureAtMinFocal",
-            "MaxApertureAtMaxFocal",
-            "MCUVersion"
-        };
+        static constexpr const char* tags[] = {
```

Comment:
Would it make sense to also convert this to a const array?

Reply:
Done.